languages/csharp/exercises/concept/expression-bodied-members/.docs/introduction.md

```diff
@@ -0,0 +1,45 @@
+Many types of struct and class members (fields being the primary exception) can use the expression-bodied member syntax. Defining a member with an expression often produces more concise and readable code than traditional blocks/statements.
+
+Methods and read-only properties are amongst the members that can be defined with expression bodies.
+
+```csharp
+public int Times3(int input) => input * 3;
+
+public int Interesting => 1729;
+```
+
+#### Ternary operators
+
+Ternary operators allow if-conditions to be defined in expressions rather than statement blocks. This echoes functional programming approaches and can often make code more expressive and less error-prone.
```

Comment:
Maybe mention that a ternary must have an if and else clause.

Reply:
Done